server_manager/www/views/server_view/server_stat_card/index.ts

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
nit: These copyright headers seem a bit off in syntax. Try using the https://github.com/google/addlicense tool:

```
~/go/bin/addlicense -c "The Outline Authors" -l apache .
```

That also helps ensure you don't miss any files (e.g. `stories.ts`).

Reply:
Thanks! Just remember to fix these `index.ts` ones.